### ISSUE TITLE
pushRepo not working properly

### ISSUE THREAD

**Issue by dturopoli** [release-it/release-it#1113]
Even though "pushRepo" option is defined it is ignored in fetch method

lib/plugin/GitBase.js

fetch(remoteUrl) {
    return this.exec('git fetch').catch(err => {
      this.debug(err);
      throw new Error(`Unable to fetch from ${remoteUrl}${EOL}${err.message}`);
    });
  }

**Comment by webpro** [comment]
Yeah, the `pushRepo` is meant to push to a different origin, not pull/fetch. You might want to use a different origin altogether to pull and push from/to the same origin?

**Comment by webpro** [comment]
Closing due to inactivity.
